jax/experimental/mesh_utils.py

```diff
@@ -171,21 +227,30 @@ def create_device_mesh(mesh_shape: Sequence[int]) -> np.ndarray:
   local_jax_devices_from_process_0 = jax.local_devices(process_index=0)
```

Comment:
nit: maybe use `process_0_devices` here too? and s/jax_devices/global devices on line 228 to be consistent?

Reply:
Done.